app/javascript/src/components/Dashboard/Notes/index.jsx

```diff
@@ -1,107 +1,66 @@
 import React, { useState, useEffect } from "react";
 
 import EmptyNotesListImage from "images/EmptyNotesList";
-import { Delete } from "neetoicons";
-import { Button, PageLoader } from "neetoui";
-import { Container, Header, SubHeader } from "neetoui/layouts";
+import { Button } from "neetoui";
+import { Container, Header } from "neetoui/layouts";
 
-import notesApi from "apis/notes";
 import EmptyState from "components/Common/EmptyState";
+import SideMenuBar from "components/Common/SideMenuBar";
+import { notes as noteList } from "data";
```

Comment:
I don't understand the benefit of this renaming. Can you please explain to me the need for doing the same? Thanks.

Reply:
@ubaidchalil Is `notes` imported from `data` a literal constant? If yes, why you have you named it in camelCase?

Reply:
Yes. But in a real scenario, the data would be camelCase. I thought like that. 
 
 I will update it to the capital snake_case.